myuw/dao/enrollment.py

```diff
@@ -28,10 +31,30 @@
 
 def get_current_quarter_enrollment(request):
     regid = get_regid_of_current_user()
+    return _get_current_quarter_enrollment(request, regid)
+
+
+def _get_current_quarter_enrollment(request, regid):
     return get_enrollment_by_regid_and_term(regid,
                                             get_current_quarter(request))
 
 
+def get_all_enrollments():
+    regid = get_regid_of_current_user()
+    return _get_all_enrollments(regid)
+
+
+def _get_all_enrollments(regid):
```

Comment:
Ethan, 
you may remove line 47-55 and use "enrollment_search_by_regid(regid)" on line 44.